utils/clients/PortfolioDbClient.ts

```diff
@@ -5,9 +5,12 @@ import {
 } from 'aws-sdk/clients/dynamodb';
 import { DynamoDbTable } from 'enums';
 import { IPortfolioClient } from 'models/clients';
-import { IPortfolio } from 'models/data';
+import * as PortfolioDefinitions from 'models/data/IPortfolio';
 import { getDynamoDbClient } from 'utils/api';
 import { isUndefined } from 'utils';
+import { CardContent, getSkeletonUtilityClass } from '@mui/material';
+import { workerData } from 'worker_threads';
+import { listConflictingAliasesMaxItemsInteger } from 'aws-sdk/clients/cloudfront';
```

Comment:
Are you using these imports? If not, they should be removed

Reply:
Will do.